### PR TITLE
TM-5960: Panel Administrator Role

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. New Menu Option for Panel Administrator (will reuse admin icon)
2. Submenu for existing panel module

_TRIPLE_ Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2959)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/471)

[Ticket](https://metaphase.atlassian.net/browse/TM-5960)

### CODE REVIEW

File: talentmap_api/user_profile/management/commands/create_seeded_users.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ class Command(BaseCommand):
         ("woodwardw", "woodwardw@state.gov", "password", "Wendy", "Woodward", False, False, ["bureau_user"]),
         ("velezp", "velezp@state.gov", "password", "Preston", "Velez", False, False, ["post_user"]),
         ("lincolna", "lincolna@state.gov", "password", "Abigail", "Lincoln", True, False, ["ao_user"]),
+        ("patricka", "patricka@state.gov", "password", "Patrick", "Anthony", False, False, ["panel_admin", "cdo"]),
```
Comment (elizabeth-jimenez) [inline talentmap_api/user_profile/management/commands/create_seeded_users.py:28]: let's add to the admin one above too. How come cdo too?

Comment (scott062) [reply talentmap_api/user_profile/management/commands/create_seeded_users.py:28]: I believe Robin mentioned all notetakers are also CDOs - I think it's one necessary level of layering in order to fetch data

Comment (elizabeth-jimenez) [reply talentmap_api/user_profile/management/commands/create_seeded_users.py:28]:
```suggestion
        ("anthonyp", "anthonyp@state.gov", "password", "Patrick", "Anthony", False, False, ["panel_admin", "cdo"]),
```

Comment (elizabeth-jimenez) [reply talentmap_api/user_profile/management/commands/create_seeded_users.py:28]: ahh gotcha!

**Comment by elizabeth-jimenez** [comment]
Changes Requested: Mock PR needed too

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**